src/utils/ast.js

```diff
@@ -2,7 +2,15 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at <http://mozilla.org/MPL/2.0/>. */
 
-export function findBestMatchExpression(symbols, tokenPos, token) {
+// @flow
+
+import type { Position } from "../types";
+import type { SymbolDeclarations } from "../workers/parser";
+
+export function findBestMatchExpression(
```

Comment:
Your fix seems more like the right one, but I'll add that the immediate thing that jumped out to me when I looked is that this function's return type is still weird. It should be `SymbolDeclaration | null` or something, but it is `SymbolDeclaration | {}` right now. That means if you were in a file that doesn't have any identifiers, it still looks like it found something.

Reply:
yeah, true